application/commands/removeValidator.go

```diff
@@ -71,33 +61,42 @@ func RemoveCommand() *cobra.Command {
 			if err != nil {
 				log.Printf("Db is already open: %v", err)
 				log.Printf("sending rpc")
+
 				var err2 error
+				validators, err2 = rpc.ShowValidators("", rpcEndpoint)
+				if err2 != nil {
+					return err2
+				}
+				if len(validators) <= 1 {
+					return errors.New(fmt.Sprintf("Cannot remove any more validators, total validators: %v", len(validators)))
+				}
+
 				validators, err2 = rpc.RemoveValidator(validatorAddress, rpcEndpoint)
 				if err2 != nil {
 					return err2
 				}
 			} else {
 				defer database.Close()
-				err = db.DeleteValidator(validatorAddress)
-				if err != nil {
-					return err
-				}
 
 				var err2 error
 				validators, err2 = db.GetValidators()
 				if err2 != nil {
 					return err2
 				}
+				if len(validators) <= 1 {
```

Comment:
this check seems to be in both if and else statements. could we extract it from if-else?

Reply:
it was added as an audit point, I think it's separate because both scenarios are based on if the orchestrator (bridge) is running or not.

Reply:
In first case the validators is being fetched using rpc (because db is locked as bridge is running), and second case directly from db (bridge is stopped)

Reply:
I see. Thank you for the clarification.